src/services/completions.ts

```diff
@@ -1322,6 +1322,12 @@ namespace ts.Completions {
             const moduleSpecifierSymbol = typeChecker.getSymbolAtLocation(importOrExportDeclaration.moduleSpecifier);
             if (moduleSpecifierSymbol) {
                 exports = typeChecker.getExportsOfModule(moduleSpecifierSymbol);
+                const exportEquals = typeChecker.resolveExternalModuleSymbol(moduleSpecifierSymbol);
+                if (exportEquals !== moduleSpecifierSymbol) {
+                    // Location doesn't matter so long as it's not an identifier.
+                    const exportEqualsType = typeChecker.getTypeOfSymbolAtLocation(exportEquals, moduleSpecifier);
+                    exports = ts.concatenate(exports, typeChecker.getPropertiesOfType(exportEqualsType));

```

Comment:
why do we need to concatenate here? how can you have exports along with `export=`?